common/src/main/java/com/redhat/cloud/notifications/models/Event.java

```diff
@@ -68,14 +66,15 @@ public class Event {
     public Event() { }
```

Comment:
Good catch. Yes. I did not want to automatically assign a new one in the constructor, as often enough it would immediately after be overwritten and only stress the GarbageCollector